contents/tutorials/public-beta-program.md

```diff
@@ -0,0 +1,279 @@
+---
+title: How to set up a public beta program using early access management
+date: 2023-07-28
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-10.png
+tags: ['feature flags']
+---
+
+Testing your new features helps make them successful. Public betas are a way to get new features into the hands of users and get valuable feedback and analytics. In this tutorial, we show you how to add a basic public beta program to your (Next.js) app with PostHog’s early access management feature.
```

Comment:
Perhaps worth mentioning that while this is Next.js, they can follow something similar for other frameworks?

```suggestion
Public betas are a way to get new features into the hands of users and get valuable feedback and analytics. In this tutorial, we show you how to add a basic public beta program to your app with PostHog’s early access management feature.

Our example in this tutorial focuses on a Next.js app. However, similar principles apply if you're using a different language or framework.
```